src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -126,31 +130,53 @@ where
             self.env.opts.periodic_compaction_interval_sec,
         ));
         min_trigger_interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Delay);
+
+        let mut min_space_reclaim_trigger_interval = tokio::time::interval(Duration::from_secs(
+            self.env.opts.periodic_space_reclaim_compaction_interval_sec,
+        ));
+        min_space_reclaim_trigger_interval
+            .set_missed_tick_behavior(tokio::time::MissedTickBehavior::Delay);
         loop {
-            let compaction_group: CompactionGroupId = tokio::select! {
-                compaction_group = sched_rx.recv() => {
-                    match compaction_group {
-                        Some(compaction_group) => compaction_group,
+            let (compaction_group, task_type) = tokio::select! {
+                recv = sched_rx.recv() => {
+                    match recv {
+                        Some((compaction_group, task_type)) => (compaction_group, task_type),
                         None => {
                             tracing::warn!("Compactor Scheduler: The Hummock manager has dropped the connection,
                                 it means it has either died or started a new session. Exiting.");
                             return;
                         }
                     }
                 },
+
                 _ = min_trigger_interval.tick() => {
                     // Disable periodic trigger for compaction_deterministic_test.
                     if self.env.opts.compaction_deterministic_test {
                         continue;
                     }
                     // Periodically trigger compaction for all compaction groups.
                     for cg_id in self.hummock_manager.compaction_group_ids().await {
-                        if let Err(e) = sched_channel.try_sched_compaction(cg_id) {
-                            tracing::warn!("Failed to schedule compaction for compaction group {}. {}", cg_id, e);
+                        if let Err(e) = sched_channel.try_sched_compaction(cg_id, compact_task::TaskType::Dynamic) {
+                            tracing::warn!("Failed to schedule base compaction for compaction group {}. {}", cg_id, e);
                         }
                     }
                     continue;
                 },
+
+                _ = min_space_reclaim_trigger_interval.tick() => {
```

Comment:
Since we does not support filter `exist_tables` in `SpaceReclamSelector`, I suggest trigger this tick in the next PR